suggestions/gocode.go

```diff
@@ -37,9 +38,13 @@ func NewGoCodeProvider(fileContainer FileContainer, gopath string) *GoCodeProvid
 
 func (p *GoCodeProvider) SuggestionsAt(runeIndex int) []gxui.CodeSuggestion {
 	cmd := exec.Command("gocode", "-f", "json", "autocomplete", p.fileContainer.Filepath(), strconv.Itoa(runeIndex))
-	cmd.Env = []string{
-		"PATH=" + os.Getenv("PATH") + string(os.PathListSeparator) + filepath.Join(p.gopath, "bin"),
-		"GOPATH=" + p.gopath,
+	if runtime.GOOS == "windows" {
```

Comment:
I would rather have two files with build tags than checking the runtime.GOOS.  The simple solution would be to add them to the `suggestions` package, e.g.

gocode.environ.windows.go
```go
// +build windows

package suggestions

func environ(gopath string) []string {
    return os.Environ()
}
```

gocode.environ.go
```go
// +build !windows

package suggestions

func environ(gopath string) []string {
    return []string{
        fmt.Sprintf("GOPATH=%s", gopath),
        fmt.Sprintf("PATH=%s%c%s", os.Getenv("PATH"), os.PathListSeparator, filepath.Join(gopath, "bin")),
    }
}
```

I think that the *best* solution would be to add an `Environ` method to the `settings.Project` type and make use of that, but that's a larger change.